### PR TITLE
liquidity: add `destaddr` parameter for autoloop loop out

### LINKED ISSUE #540: Request: Autoloop to accept `addr` parameter
Loop out supports the --addr parameter to directly loop out to another address.  This is very useful for our specific use case and I've been using loop out very happily on the CLI

I was really hoping to use autoloop, but unfortunately since it doesn't support a destination address, it would require two on-chain transactions to get to the right destination.

Is it possible to enable autoloop/liquidity manager to accept the following: `loop setparams --addr=XXX` ?

Two comments:

1. This would be a game changer for my use case (currently requires manual admin every day), and I would definitely use autoloop if this is enabled
2. I don't need to change the address often (possibly at all)

Or is there some alternative/workaround that I have not thought about?

### PR DESCRIPTION
**Pull request by GeorgeTsagk**
# Description

This PR introduces a parameter for autoloop that lets the user define a custom address to be used for loop out.

Closes #540

### CODE REVIEW

**Review by ckritzinger — APPROVED** [review]

File: cmd/loop/liquidity.go [hunk 1]
```diff
@@ -286,6 +286,11 @@ var setParamsCommand = cli.Command{
 				"of swaps, limited to the budget set by " +
 				"autobudget",
 		},
+		cli.StringFlag{
+			Name: "outaddr",
```
Comment (ckritzinger) [inline cmd/loop/liquidity.go:290]: nit: for the loop command, this same parameter is "--addr", I don't know how much the consistency matters to you

File: looprpc/client.proto [hunk 2]
```diff
@@ -854,6 +854,11 @@ message LiquidityParameters {
     The confirmation target for loop in on-chain htlcs.
     */
     int32 htlc_conf_target = 17;
+
+    /*
+    The address to use for autoloop loop out HTLCs.
```
Comment (bhandras) [inline looprpc/client.proto:859]: nit: do you mean for the the destination address?

**Review by bhandras — APPROVED** [review]
Changes LGTM :tada: 

Although the changes are trivial, some basic test coverage would still be nice.

File: looprpc/client.proto (hunk 2 above)
Comment (GeorgeTsagk) [reply looprpc/client.proto:859]: Yes, the address where the funds are going to after we sweep the HTLC

**Review by sputn1ck — COMMENTED** [review]
LGTM, but would also vouch for more consistent naming. Do you plan on adding tests?

**Comment by GeorgeTsagk** [comment]
> LGTM, but would also vouch for more consistent naming. Do you plan on adding tests?

Yes, will drop them later today.

**Review by bhandras — APPROVED** [review]
Re-reviewed with the added test coverage, LGTM :tada:

**Review by sputn1ck — APPROVED** [review]
LGTM. I'm still of the opinion for the consistent naming though 😅

**Comment by GeorgeTsagk** [comment]
Changed parameter name to `destaddr` and changed codebase variables accordingly.

**Comment by GeorgeTsagk** [comment]
Rebased on master following @sputn1ck 's CI fix

**Pull request merged**